todo_app/data/trello_items.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+from telnetlib import STATUS
+from turtle import update
+import requests
```

Comment:
Note that you haven't tracked this anywhere in your project, so I'll need to independently remember to add this library; not an issue for one file but a pain if there are lots! Instead we should make sure we're tracking all of our dependencies with poetry (by running `poetry add requests`) which means future users will pick up this dependency automatically by running `poetry install`

Reply:
I have removed them. Not sure where they came from :)